server/filestore_test.go

```diff
@@ -3539,3 +3539,50 @@ func TestFileStoreFetchPerf(t *testing.T) {
 	}
 	fmt.Printf("Elapsed to load all messages is %v\n", time.Since(now))
 }
+
+func TestFileStoreSubjectInfoWithSnapshotCleanup(t *testing.T) {
```

Comment:
Ran that test on -race mode and it takes about 150MB (only 29MB without -race). I think it is ok, but just letting you know (I noticed the 10,000 msgs of 1K each).

Reply:
Should I put under norace?

Reply:
Maybe move.

Reply:
Done.